src/ui/preferences.ui

```diff
@@ -336,32 +277,290 @@ be the last ones.</string>
+         <property name="flat">
+          <bool>false</bool>
+         </property>
+         <property name="checkable">
+          <bool>true</bool>
+         </property>
+         <property name="checked">
+          <bool>false</bool>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout">
+          <item>
+           <widget class="QCheckBox" name="startToTray">
+            <property name="toolTip">
+             <string>With this on, the application starts directly to system tray without showing
+its main window.</string>
+            </property>
+            <property name="text">
+             <string>Start to system tray</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QCheckBox" name="closeToTray">
+            <property name="toolTip">
+             <string>With this on, an attempt to close main window would hide it instead of closing
+the application.</string>
+            </property>
+            <property name="text">
+             <string>Close to system tray</string>
+            </property>
+           </widget>
+          </item>
+         </layout>
+        </widget>
        </item>
       </layout>
      </widget>
+     <widget class="QWidget" name="tab_font">
+      <property name="maximumSize">
+       <size>
+        <width>763</width>
+        <height>16777215</height>
+       </size>
+      </property>
+      <attribute name="icon">
+       <iconset resource="../../resources.qrc">
+        <normaloff>:/icons/font.svg</normaloff>:/icons/font.svg</iconset>
+      </attribute>
+      <attribute name="title">
+       <string>Font</string>
+      </attribute>
+      <widget class="QGroupBox" name="groupBox_5">
+       <property name="geometry">
+        <rect>
+         <x>1</x>
+         <y>1</y>
+         <width>415</width>
+         <height>100</height>
+        </rect>
+       </property>
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="minimumSize">
+        <size>
+         <width>0</width>
+         <height>100</height>
+        </size>
+       </property>
+       <property name="title">
+        <string>Standard Font</string>
+       </property>
+       <layout class="QVBoxLayout" name="verticalLayout_24">
+        <item>
+         <widget class="QFontComboBox" name="font_standard">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QLabel" name="text_standard">
+          <property name="text">
+           <string>GoldenDict  next generation dictionary,複数のフォーマットのサポート</string>
```

Comment:
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/10b471c0-03da-45fb-bb72-9da78457f2f9)

Reply:
I feel like half of the panel is enough. The options on the left don't have to be changed. :sweat_smile: 

We can group the original options as "Appearance" so that they can be layoutted in parallel

![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/45d12459-4ef3-42b7-92eb-49904d909109)

Reply:
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/edd508d2-95c6-4058-9237-a34e3f1ffbb3)
what about this

Reply:
I prefer this so that we can expand "Appearances" in future.
![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/4264b62e-c753-4cd8-859b-3608b872babd)

Reply:
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/b13d054d-687d-40da-85b3-915042b0f930)

Reply:
The original options were on the left, I think it would be better if we don't break the positioning :sweat_smile:

Reply:
> I prefer this so that we can expand "Appearances" in future. ![image](https://user-images.githubusercontent.com/20123683/253788948-4264b62e-c753-4cd8-859b-3608b872babd.png)

changed.